variables.tf

```diff
@@ -267,3 +267,21 @@ variable "additional_databases" {
   }))
   default = []
 }
+
+variable "maintenance_window" {
+  description = <<-EOT
+  day: The day of week (1-7), starting on Monday.
+  hour: The hour of day (0-23), ignored if day not set.
+  update_track: The update track of maintenance window. Can be either `canary` or `stable`.
+  EOT
+  type = object({
+    day_utc      = number
+    hour_utc     = number
+    update_track = string
+  })
+  default = {
+    day_utc      = 1  # implies Tuesday in MYT
+    hour_utc     = 19 # implies 3AM in MYT
+    update_track = "canary"
```

Comment:
By default, we should recommend `stable`

Reply:
This config is useful when there is more than one SQL instance in the same project. Lets say one for staging and another for production.
The config indicates to receive updates earlier (canary) or later (stable). So you would ideally set `canary` for staging and `stable` for production. So staging DB will get updated first before production DB update.

According to [GCP docs](https://cloud.google.com/sql/docs/mysql/instance-settings?hl=en#maintenance-timing-2ndgen):
> The relative timing of updates is not observed between projects; if you have instances with an earlier timing setting in a different project than your instances with a later timing setting, Cloud SQL makes no attempt to update the instances with the earlier timing setting first.